actions/action-lint/README.md

```diff
@@ -0,0 +1,46 @@
+# helm-lint
+
+This action will lint all actions and workflows of a repository.
+
+## Prerequisites
+
+You need to create the lint matcher file `actionlint-matcher.json` and configure it to your liking.
+The default matcher is the following:
+
+```json
+{
+  "problemMatcher": [
+    {
+      "owner": "actionlint",
+      "pattern": [
+        {
+          "regexp": "^(?:\\x1b\\[\\d+m)?(.+?)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*: (?:\\x1b\\[\\d+m)*(.+?)(?:\\x1b\\[\\d+m)* \\[(.+?)\\]$",
+          "file": 1,
+          "line": 2,
+          "column": 3,
+          "message": 4,
+          "code": 5
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
customization is not necessary. We will always use the default problem matcher from here https://raw.githubusercontent.com/rhysd/actionlint/main/.github/actionlint-matcher.json

you can remove this entire paragraph